src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -237,16 +218,14 @@ PMComplexTest >> testComplexNumberAndIntegerAreUnequalEvenIfRealPartIsEqualToInt
 	self deny: 1 + 3 i equals: 1.
 ]
 
-{ #category : #tests }
+{ #category : #'testing - arithmetic' }
 PMComplexTest >> testConversion [
-	"self run: #testConversion"
-
-	"self debug: #testConversion"
-
 	self assert: 1 + 2 i + 1 equals: 2 + 2 i.
 	self assert: 1 + (1 + 2 i) equals: 2 + 2 i.
+	
```

Comment:
I've wondered whether we might add separate tests to demonstrate **commutativity** and **associativity** of addition. I think it is the definition of a (mathematical) field. Similarly, we might demonstrate the commutativity of multiplication, another part of the definition of a mathematical field.

@olekscode @SergeStinckwich @khinsen what are your thoughts?. This could become a contract test for all fields, where addition and multiplication are commutative and associative.

Reply:
I'm not sure I find this test useful. Does anyone else feel different?

Reply:
Yes, that could be a good idea
For example:
```st
testAddition
    self assert: 2 + 3 equals: 5.

testAdditionIsCommutative
    self assert: 2 + 3 equals: 3 + 2.
```
And so on